src/opm/output/eclipse/AggregateMSWData.cpp

```diff
@@ -181,29 +181,35 @@ namespace {
     }
 
     Opm::RestartIO::Helpers::SegmentSetSourceSinkTerms
-    getSegmentSetSSTerms(const std::string& wname, const Opm::WellSegments& segSet, const std::vector<Opm::data::Connection>& rateConns,
-                         const Opm::WellConnections& welConns, const Opm::UnitSystem& units)
+    getSegmentSetSSTerms(const Opm::WellSegments& segSet,
```

Comment:
Maybe two words about what the `SourceSinkTerms` constitute?

Reply:
I'll add a comment.